remo/reports/models.py

```diff
@@ -320,23 +318,28 @@ def create_update_passive_event_report(sender, instance, created, **kwargs):
                                     instance.region,
                                     instance.country),
         'link': get_event_link(instance),
-        'activity_description': instance.description}
+        'activity_description': instance.description,
+        'is_passive': True,
+        'event': instance,
+        'campaign': instance.campaign
+    }
 
     if created:
         activity = Activity.objects.get(name=ACTIVITY_EVENT_CREATE)
-        attrs.update({
-            'user': instance.owner,
-            'event': instance,
-            'activity': activity,
-            'is_passive': True})
+        attrs.update({'user': instance.owner,
+                      'activity': activity})
```

Comment:
If we move this to line 324, then when updating it's going to overwrite all the activities in the reports with this one.